examples/returns-from-sap/.devcontainer.json

```diff
@@ -0,0 +1,4 @@
+{
```

Comment:
Unnecessary file

Reply:
Updated in [Add reviewed changes and documentation](https://github.com/ballerina-platform/module-ballerinax-sap.s4hana.delivery/pull/15/commits/918d0df8497a939d2833b9ed1b4509b83f1f89c0)